Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Christopher Loggins</authors>
+        <owners>Christopher Loggins</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.0" />
```

Comment:
I'd feel more comfortable if this were a version range, with 2.0 being the inclusive minimum and 3.0 being the exclusive maximum.

We won't be pushing out 3.0 any time soon, but we will be doing our best guarantee that the 2.x version range is non-breaking.